context/app/static/js/components/Routes/Route/Route.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { PropsWithChildren, Suspense } from 'react';
+import Stack from '@mui/material/Stack';
+import Box from '@mui/material/Box';
+import { ContainerProps } from '@mui/material/Container';
+
+import { useIsDesktop } from 'js/hooks/media-queries';
+import RouteLoader from '../RouteLoader';
+import { StyledContainer } from './style';
+
+export const leftRouteBoundaryID = 'left-route-boundary';
+export const rightRouteBoundaryID = 'right-route-boundary';
+
+function RouteBoundary({
+  id,
+  showBoundary,
+}: {
+  id: typeof leftRouteBoundaryID | typeof rightRouteBoundaryID;
+  showBoundary: boolean;
+}) {
+  return <Box id={id} flex="1 0" sx={{ padding: 2, ...(!showBoundary && { display: 'none' }) }} />;
```

Comment:
I think we can provide the padding/display properties directly here since they're system props